daemon/algod/api/algod.oas2.json

```
@@ -3564,6 +3608,25 @@
         }
       }
     },
+    "BoxReference": {
+      "description": "References a box of an application.",
+      "type": "object",
+      "required": [
+        "app",
+        "name"
+      ],
+      "properties": {
+        "app": {
+          "description": "App ID which this box belongs to",
```

Moving this suggestion to the actual source file.
```suggestion
          "description": "Application ID which this box belongs to",
```

---

Done in e423c7d2f